sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_constants.py

```diff
@@ -31,7 +31,7 @@
 METRIC_EXPORTER_NAMES_ARG = "metric_exporter_names"
 SAMPLER_ARG = "sampler"
 TRACE_EXPORTER_NAMES_ARG = "trace_exporter_names"
-
+APPLICATIONINSIGHTS_PREVIEW_LIVE_METRICS_ENABLED = "APPLICATIONINSIGHTS_PREVIEW_LIVE_METRICS_ENABLED"
```

Comment:
Nit: Should end with "...ARG" to be consistent with the other config names.

Reply:
The other ones are keys to the config map, i.e. argument names, such as ENABLE_LIVE_METRICS_ARG.

This is just an env var. It just looks odd because all the other env vars are either imported from OT or stored in the exporter. So this is the only of its kind. So far, our style for env vars is either X, _X, or X_ENV_VAR. Would you prefer